gui/src/components/modalServer.vue

```diff
@@ -956,11 +973,12 @@ export default {
           id: decodeURIComponent(u.username),
           net: u.params.type || "tcp",
           type: u.params.headerType || "none",
-          host: u.params.sni || u.params.host || "",
+          host: u.params.host || "",
```

Comment:
Some share-links have no sni field and obey the old standard. Can we figure out how to be compatible with them?

Reply:
Do you want me to remove host from it then we only should have it from settings or none?

Reply:
It's okay, I'll fix it.